addons/binding/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/SomfyTahomaBindingConstants.java

```diff
@@ -148,7 +148,7 @@
     public static final String TAHOMA_URL = API_URL + "externalAPI/json/";
     public static final String TAHOMA_EVENT_URL = API_URL + "enduserAPI/events/";
     public static final String SETUP_URL = API_URL + "enduserAPI/setup/gateways/";
-    public static final String REFRESH_URL = API_URL + "enduserAPI/setup/devices/states/refresh";
+    public static final String REFRESH_URL = TAHOMA_URL + "refreshAllStates";
```

Comment:
Does this also work for existing devices?

Reply:
Yes, it works for both gateways Tahoma and Connexoon.